init.lua

```diff
@@ -54,10 +59,28 @@ local function check_value(table, value)
     return false
 end
 
+--- Verify that the request was sent from GitHub by validating SHA256.
+-- Returns `true` if the request is authorized, otherwise `false`.
+-- @see https://docs.github.com/en/webhooks-and-events/webhooks/securing-your-webhooks
+local function is_verified_signature(body, signature)
+    if not signature then
+        return false
+    end
+    local sign = 'sha256=' .. crypto.hmac.sha256_hex(github_signature, body)
+    if utf8.cmp(signature, sign) == 0 then
+        return true
+    end
+    return false
+end
+
 --- Handle an incoming webhook from GitHub.
 -- @see https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks
 function webhook_handler(req)
-    local payload = req:json()
+    local body = req:read()
+    if not is_verified_signature(body, req.headers['x-hub-signature-256']) then
+        return {status = 403}
```

Comment:
Please add a note to README.md about this status code.

Reply:
Yep, added.